src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/GeneralLetterService.java

```diff
@@ -105,8 +105,8 @@ private void populateNameAddressAndReference(CaseDetails caseDetails) {
                 .name((String) data.get(RESP_SOLICITOR_NAME))
                 .formattedAddress(documentHelper.formatAddressForLetterPrinting((Map) data.get(RESP_SOLICITOR_ADDRESS)));
         } else if ("respondent".equalsIgnoreCase(generalLetterAddressTo)) {
-            String respondentFmNameCcdField = isConsentedApplication ? APP_RESPONDENT_FIRST_MIDDLE_NAME : "respondentFMName";
-            String respondentLastNameCcdField = isConsentedApplication ? APP_RESPONDENT_LAST_NAME : "respondentLName";
+            String respondentFmNameCcdField = isConsentedApplication ? CONSENTED_RESPONDENT_FIRST_MIDDLE_NAME : "respondentFMName";
```

Comment:
use the constants fam

Reply:
These were a result of renaming an existing constant, I will add the other field constants in here. Good spot